### PR TITLE
feat(sonic): restructuring page

### PR DESCRIPTION
**Pull request by akristen**
- [x] Made some changes based on peer reviews. 
- [x] Uncertain about when users should want to configure the infra agent for log forwarding (do they need to worry about the configuration steps if they did guided install?), so waiting to merge until I get another check.
- [x] Get final peer review on guided install section + the second install flow for manual install

### CODE REVIEW

File: src/content/docs/logs/forward-logs/forward-your-logs-using-infrastructure-agent.mdx [hunk 1]
```diff
@@ -618,7 +491,7 @@ What you use for the log source will depend on where you want to forward your lo
 
 ### Optional configuration [#optional-config]
 
-The following configuration parameters are not required but are still recommended.
+Although these configuration parameter's aren't required, we still recommend you set them up so you get the most functionality out of log forwarding.
```
Comment (akristen) [inline src/content/docs/logs/forward-logs/forward-your-logs-using-infrastructure-agent.mdx:494]:
```suggestion
Although these configuration parameters aren't required, we still recommend you set them up so you get the most functionality out of log forwarding.
```

File: src/content/docs/logs/forward-logs/forward-your-logs-using-infrastructure-agent.mdx [hunk 2]
```diff
@@ -34,35 +34,9 @@ import infrastructureWindows from 'images/infrastructure_logo_windows.png'
...
-1. If you haven't already, [create a New Relic account](https://newrelic.com/signup). It's free, forever.
-2. Verify the [system requirements](#requirements) needed for configuring logs.
-3. Ensure you have [installed the infrastructure agent](#install), version 1.11.4 or higher. (If you use the guided install process in the UI, skip the following steps.)
-4. Create a `logging.yml` [configuration file](#configure) in the infrastructure agent's `logging.d` directory.
-5. [Configure](#parameters) your log sources and other parameters.
-6. Generate some traffic and wait a few minutes, then [check your account](#find-data) for data.
-7. [Explore your log data](#what-next) in the logs UI. In New Relic, look for log data that's been reported by the infrastructure agent.
-
-Here's an example of logs for your host's UI. You can see logs in context of events for the selected time period, and drill down into detailed data for any of the highlighted attributes. To examine even more detailed data, run a query, or click **Open in logs**.
+Your first step is to [create a New Relic account](). After that, use this doc to set up log forwarding.
```
Comment (jeff-colucci) [inline src/content/docs/logs/forward-logs/forward-your-logs-using-infrastructure-agent.mdx:39]: Not sure if the link is supposed to be missing here, just doing due diligence so we have a marker.

File: src/content/docs/logs/forward-logs/forward-your-logs-using-infrastructure-agent.mdx [hunk 3]
```diff
@@ -71,87 +45,19 @@ Here's an example of logs for your host's UI. You can see logs in context of eve
...
 
-## Enable logging for your on-host integrations [#on-host]
 
-With the infrastructure agent installed, you can enable automatic log parsing and forwarding for our most popular on-host integrations with one step. To enable this feature, rename the `on-host-log.yml.example` file to `on-host-log.yml`. Once done, your integration's logs are automatically parsed and sent to New Relic.
+## Install the infrastructure agent [#infra]
 
-This option is available for our [supported Linux platforms](#requirements).
-
-To enable the on-host integration log forwarding feature:
+Starting with version 1.11.4, the infrastructure agent forwards logs to New Relic. You can install the infrastructure agent with our guided install, or with a manual install. Our guided install is a lightweight route that installs our infrastructure agent, an APM agent, and more. This is a good choice for smaller organizations or any organization that wants to test out New Relic.
```
Comment (jeff-colucci) [inline src/content/docs/logs/forward-logs/forward-your-logs-using-infrastructure-agent.mdx:54]: Do we need to put the version where this feature started here? Just asking for curiosity's sake

Comment (jeff-colucci) [reply src/content/docs/logs/forward-logs/forward-your-logs-using-infrastructure-agent.mdx:54]: I don't think you need a comma in the second sentence. Something like "guided or manual install" might work better and eliminates minor repetition.

File: src/content/docs/logs/forward-logs/forward-your-logs-using-infrastructure-agent.mdx [hunk 4]
```diff
@@ -314,31 +213,15 @@ To use the following links, make sure you are logged to your New Relic account.
...
 
-<InstallFeedback />
+<Callout variant="important">
+  The log forwarding feature is not included when the infrastructure agent is implemented using Linux tarball or Windows ZIP installations.
+</Callout>
 
 ## Configure the infrastructure agent [#configure]
 
-Configuration files describe which log sources are forwarded. Our infrastructure agent uses `.yml` files to configure logging. You can add as many config files as you want.
+Configuration files direct your system to forward the log sources you want to appear in New Relic. You can add as many config files as you want. Our infrastructure agent uses `.yml` files to configure logging. If you install the infrastructure agent via [Add data](https://one.newrelic.com/marketplace?state=78678a7f-91c5-ca40-ac55-e6b74a50085c) in the UI, the file `logging.yml` is created automatically.
```
Comment (jeff-colucci) [inline src/content/docs/logs/forward-logs/forward-your-logs-using-infrastructure-agent.mdx:224]: I like this change!

File: src/content/docs/logs/forward-logs/forward-your-logs-using-infrastructure-agent.mdx [hunk 5]
```diff
@@ -349,23 +232,13 @@ To add a new configuration file for the log forwarding feature:
...
-    <Callout variant="tip">
-      If you install the infrastructure agent via [Add data](https://one.newrelic.com/marketplace?state=78678a7f-91c5-ca40-ac55-e6b74a50085c) in the UI, the file `logging.yml` is created automatically.
-    </Callout>
-
 The agent automatically processes new configuration files without having to restart the infrastructure monitoring service. The only exception to this is when configuring a custom Fluent Bit configuration.
 
 ## Log forwarding parameters [#parameters]
 
-The infrastructure log forwarding `.yml` config supports the following parameters:
+You're required to set up a `name` and log source parameter in the log forwarding `.yml` config file. To start, define a `name` of the log or logs you want to forward to New Relic.
```
Comment (jeff-colucci) [inline src/content/docs/logs/forward-logs/forward-your-logs-using-infrastructure-agent.mdx:239]: Feels a bit disjointed to have name in code but log source parameter not. Not sure if the fix is both being in code format or neither.

File: src/content/docs/logs/forward-logs/forward-your-logs-using-infrastructure-agent.mdx [hunk 6]
```diff
@@ -878,6 +751,101 @@ If everything is configured correctly and your data is being collected, you shou
...
+
+    ```
+    sudo cp /etc/newrelic-infra/logging.d/redis-log.yml.example /etc/newrelic-infra/logging.d/redis-log.yml
+    ```
+  </Collapser>
+</CollapserGroup>
+
+## Enable log forwarding on agent installed using Linux tarball [#tarball-install]
+
+Our custom Linux installation process for infrastructure monitoring allows you to tailor all aspects of the installation process, and to place files and folders on your machine. If you choose the [assisted](https://docs.newrelic.com/docs/infrastructure/install-infrastructure-agent/linux-installation/tarball-assisted-install-infrastructure-agent-linux) or [manual](https://docs.newrelic.com/docs/infrastructure/install-infrastructure-agent/linux-installation/tarball-manual-install-infrastructure-agent-linux) tarball installation process, follow these steps to implement the log forwarder feature:
```
Comment (jeff-colucci) [inline src/content/docs/logs/forward-logs/forward-your-logs-using-infrastructure-agent.mdx:831]: I think the links are formatted incorrectly but I'm not sure if they need to be full or start at /docs/ for some specific reason.

File: src/content/docs/logs/forward-logs/forward-your-logs-using-infrastructure-agent.mdx [hunk 7]
```diff
@@ -878,6 +751,101 @@ If everything is configured correctly and your data is being collected, you shou
...
+- `/etc/newrelic-infra/logging.d`
+
+2. Download and install New Relic's [fluent-bit-package (RPM)](https://github.com/newrelic/fluent-bit-package/releases) by running a command similar to:
+  ```shell
+   yum localinstall td-agent-bit-<some-version>.rpm`
+   ```
+
+3. Download New Relic's [fluentbit plugin](https://github.com/newrelic/newrelic-fluent-bit-output/releases) and save it as `/var/db/newrelic-infra/newrelic-integrations/logging/out_newrelic.so`.
+
+4. Dowload or copy the `parsers.conf` file from [this Github repository](https://github.com/newrelic/fluent-bit-package/blob/main/parsers.conf), and save it as `/var/db/newrelic-infra/newrelic-integrations/logging/parsers.conf`.
```
Comment (jeff-colucci) [inline src/content/docs/logs/forward-logs/forward-your-logs-using-infrastructure-agent.mdx:845]: Download misspelled. Should Vale have caught this?

**Review by jeff-colucci — COMMENTED** [review]
Just saw a couple of things but overall looks great!

File: src/content/docs/logs/forward-logs/forward-your-logs-using-infrastructure-agent.mdx (hunk 2 above)
Comment (akristen) [reply src/content/docs/logs/forward-logs/forward-your-logs-using-infrastructure-agent.mdx:39]: LOL omg thank you.

File: src/content/docs/logs/forward-logs/forward-your-logs-using-infrastructure-agent.mdx [hunk 8]
```diff
@@ -349,23 +232,13 @@ To add a new configuration file for the log forwarding feature:
 
 2. Create a `logging.yml` configuration file, and add the parameters you need. The `logging.d` directory has various `.yml.example` files you can use as a reference or starting point.
 
-    <Callout variant="tip">
-      If you install the infrastructure agent via [Add data](https://one.newrelic.com/marketplace?state=78678a7f-91c5-ca40-ac55-e6b74a50085c) in the UI, the file `logging.yml` is created automatically.
-    </Callout>
-
 The agent automatically processes new configuration files without having to restart the infrastructure monitoring service. The only exception to this is when configuring a custom Fluent Bit configuration.
 
 ## Log forwarding parameters [#parameters]
```
Comment (akristen) [inline src/content/docs/logs/forward-logs/forward-your-logs-using-infrastructure-agent.mdx:237]: Maybe more action oriented title

File: src/content/docs/logs/forward-logs/forward-your-logs-using-infrastructure-agent.mdx [hunk 9]
```diff
@@ -34,35 +34,9 @@ import infrastructureWindows from 'images/infrastructure_logo_windows.png'
 
 import osAnsibleRed from 'images/os_icon_ansible-red.png'
 
-One option for [reporting your logs to New Relic](/docs/logs/get-started/get-started-log-management#integrate-logs) is using our infrastructure monitoring agent. This makes all of your logging data available in one location and provides deeper visibility into both your application and your platform performance data.
+Forwarding your logs to New Relic makes all of your logging data available in one location, providing deeper visibility into both your application and your platform performance data. With your logs in one spot, you get enhanced log management capabilities that let you collect, process, explore, query, and alert on errors or anomalies found in your log data.
```
Comment (akristen) [inline src/content/docs/logs/forward-logs/forward-your-logs-using-infrastructure-agent.mdx:37]:
```suggestion
Forwarding your logs to New Relic makes all of your logging data available in one location, providing deeper visibility into both your application and your platform performance data. With your logs in one spot, you can collect, process, explore, query, and alert on errors or anomalies found in your log data. 
```

File: src/content/docs/logs/forward-logs/forward-your-logs-using-infrastructure-agent.mdx [hunk 10]
```diff
@@ -71,198 +43,42 @@ Here's an example of logs for your host's UI. You can see logs in context of eve
...
-
-* Infrastructure agent version 1.11.4 or higher.
+* Infrastructure agent version 1.11.4 or higher
 * [Fluent Bit](https://fluentbit.io/). The infrastructure agent already installs the latest version for you. To update or downgrade it to a specfic version, refer to the [Fluent Bit installation](#install-fb-version) procedures.
-* OpenSSL library 1.1.0 or higher is required by the infrastructure agent starting from version 1.16.4.
+* OpenSSL library 1.1.0 or higher
 * Built-in support for ARM64 architecture on Linux systems (for example, AWS Graviton architecture) added in infrastructure agent [1.20.6](https://github.com/newrelic/infrastructure-agent/releases/tag/1.20.6).
+* Amazon Linux 2
+* CentOS version 7 or higher
+* Debian version 9 ("Stretch") or higher. Version 11 is not supported.
```
Comment (jeff-colucci) [inline src/content/docs/logs/forward-logs/forward-your-logs-using-infrastructure-agent.mdx:68]: This isn't a big deal at all I don't think but the period makes the punctuation inconsistent. Just doing due diligence.

**Review by jeff-colucci — APPROVED** [review]
Only saw one tiny thing but I'm going to go ahead and approve, everything else looks good

**Review by jeff-colucci — APPROVED** [review]
Approve!

**Pull request merged**